consensus/istanbul/backend.go

```diff
@@ -97,9 +99,67 @@ type Backend interface {
 	// ParentBlockValidators returns the validator set of the given proposal's parent block
 	ParentBlockValidators(proposal Proposal) ValidatorSet
 
-	// RefreshValPeers will connect with all the validators in the validator connection set and disconnect validator peers that are not in the set
-	RefreshValPeers() error
-
 	// Authorize injects a private key into the consensus engine.
 	Authorize(address common.Address, publicKey *ecdsa.PublicKey, decryptFn DecryptFn, signFn SignerFn, signBLSFn BLSSignerFn)
 }
+
+// BackendForProxy provides the Istanbul backend application specific functions for Istanbul proxy
+type BackendForProxy interface {
+	// Address returns the owner's address
+	Address() common.Address
+
+	// IsProxiedValidator returns true if this node is a proxied validator
```

Comment:
It seems like the functions needed from a Validator and the functions needed from a Proxy are somewhat different, with quite a bit of overlap. Maybe we should split this into two interfaces (e.g. a `ValidatorBackend` interface and a `ProxyBackend` interface)

Reply:
Done.